h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java

```diff
@@ -683,6 +666,14 @@ private void parsePropertiesMetadata(JsonParser jp) throws IOException, SQLExcep
                 case VALUE_NUMBER_INT:
                     cachedColumnNames.put(fieldName, "BIGINT");
                     break;
+                case START_ARRAY:
+                    cachedColumnNames.put(fieldName, "VARCHAR");
```

Comment:
Not sure 'VARCHAR' is the best type to use. You should have searched on databases data type to find the best one : [postgres](https://www.postgresql.org/docs/9.3/static/datatype.html) and [H2](http://h2database.com/html/datatypes.html)